ui_framework/components/global_styles/variables/_z_index.scss

```diff
@@ -0,0 +1,5 @@
+$kuiZContent:         1;
+$kuiZContentMenu:     100;
+$kuiZNavigation:      200;
+$kuiZMask:            300;
+$kuiZModal:           400;
```

Comment:
How about increments of `1000`? It doesn't hurt and gives us more room to expand in the future.